src/lib/services/playground/admin.ts

```diff
@@ -162,7 +162,7 @@ export const setApplicationStatus = ({
             ['Twitter', updatedApplication.twitterUrl],
             ['Instagram', updatedApplication.instagramUrl],
             ['LinkedIn', updatedApplication.linkedinUrl],
-            ['Message', updatedApplication.moreInfo],
+            ['Message', updatedApplication.moreInfo.split('\n').join('<br/>')],
```

Comment:
i think it would be better if instead of splitting, which creates an array with multiple strings in memory and then joining again, a replace expression would be more suitable. you could also catch trailing \r line endings transmitted by microsoft browsers, like edge.
example:
```typescript
['Message', updatedApplication.moreInfo.replace(/\r?\n/g, "<br>")],
```
further it would be desireble to limit the horizontal space any of these fields can take up by setting appropriate styling on the container surrounding them.